docs.css

```diff
@@ -0,0 +1,106 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Montserrat", sans-serif;
+}
+
+.carbon {
+  width: 100%;
+  margin-bottom: 3rem;
+}
+/* :root{
+    --primary-color:#3B82F6;
+    --secondary-color:#6B7280;
+    --white-color:#E5E7EB;
+    --grey: rgb(206, 206, 206)
+} */
+iframe {
+  width: 85% !important;
+  min-height: 40vh;
+  border: 0;
+  transform: scale(1);
+  overflow: hidden;
+  display: block !important;
+}
+body {
+  display: grid;
+  grid-template-rows: rem 20fr;
+}
+.home-details {
+  text-align: center;
+  width: 85%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 20vh;
+}
+
+main {
+  display: grid;
+  grid-template-columns: 1fr 4fr;
+  border-top: none !important;
+}
+.sidebar {
+  border: 1px solid black;
+  background-color: whitesmoke;
+  border-right: none;
+  border-top: none !important;
+}
+.sidebar-item {
+  padding: 0.8rem 1.5rem;
+  font-size: 1.8rem;
+  font-weight: bolder;
+  /* margin-left: 1.6rem; */
+  border-left: 0.8rem solid grey;
+  list-style: none;
+  cursor: pointer;
+  text-decoration: none;
+  border-right: 1px solid black;
+}
+.sidebar-item:hover {
+  background-color: rgb(255, 255, 255);
+  border-right: none;
+  border-left: 1rem solid grey;
+  transition: 0.4s;
+}
+a {
+  text-decoration: none;
+  color: black;
+}
```

Comment:
declaring a class for `<a>` would be better instead of directly changing the style of element